Debian Bullseye: Python2 still required

With https://github.com/mxe/mxe/commit/3b61050dabf8b3fb0fae9e373ebb88205758ced0 the list of required packages for Debian changed.  It no longer lists python (i.e. Python 2) but only Python 3. It seems that Python 2 is still required, though.  Tested using a docker image:


```dockerfile
FROM debian:bullseye

# First `apt-get install` is the list from
# <https://mxe.cc/#requirements-debian>.
RUN apt-get update && apt-get upgrade -y && \
    apt-get install -y --no-install-recommends \
        autoconf \
        automake \
        autopoint \
        bash \
        bison \
        bzip2 \
        flex \
        g++ \
        g++-multilib \
        gettext \
        git \
        gperf \
        intltool \
        libc6-dev-i386 \
        libgdk-pixbuf2.0-dev \
        libltdl-dev \
        libssl-dev \
        libtool-bin \
        libxml-parser-perl \
        lzip \
        make \
        openssl \
        p7zip-full \
        patch \
        perl \
        python3 \
        python3-mako \
        ruby \
        sed \
        unzip \
        wget \
        xz-utils && \
    apt-get autoremove -y

# Download MXE
RUN mkdir /build
WORKDIR /build
RUN git clone https://github.com/mxe/mxe.git

RUN cd mxe && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 cc zlib gnutls \
           && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 boost \
           && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 mesa \
           && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 qtbase \
           && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 qttools \
                 qtimageformats \
                 qtmultimedia \
                 qtquickcontrols \
                 qtquickcontrols2 \
            && make clean-junk \
            && rm -rf /build/mxe/src /build/mxe/log /build/mxe/pkg /build/mxe/.git /build/mxe/.ccache

# Enhance path
ENV PATH /build/mxe/usr/bin:$PATH

# Add a qmake aliases
RUN ln -s /build/mxe/usr/bin/x86_64-w64-mingw32.shared-qmake-qt5 /build/mxe/usr/bin/qmake

RUN mkdir /src
WORKDIR /src
```

This is just an example. The relevant log output is:

```
Failed to build package mxe-conf for target x86_64-pc-linux-gnu!
------------------------------------------------------------
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
python --version
bash: line 1: python: command not found
make[1]: *** [Makefile:888: build-only-mxe-conf_x86_64-pc-linux-gnu] Error 127
make[1]: Leaving directory '/build/mxe'
real    0m1.968s
user    0m1.889s
sys     0m0.092s
```

Or longer:

```
Step 6/10 : RUN cd mxe && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 cc zlib gnutls            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 boost            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 mesa            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 qtbase            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 qttools                  qtimageformats                  qtmultimedia                  qtquickcontrols                  qtquickcontrols2             && make clean-junk             && rm -rf /build/mxe/src /build/mxe/log /build/mxe/pkg /build/mxe/.git /build/mxe/.ccache
 ---> Running in 4f7e36687d08
bash: - : invalid option
Usage:  bash [GNU long option] [option] ...                                                                                                                   
        bash [GNU long option] [option] script-file ...
GNU long options:
        --debug
        --debugger
        --dump-po-strings
        --dump-strings
        --help
        --init-file
        --login
        --noediting
        --noprofile
        --norc
        --posix
        --pretty-print
        --rcfile
        --restricted
        --verbose
        --version
Shell options:
        -ilrsD or -c command or -O shopt_option         (invocation only)
        -abefhkmnptuvxBCHP or -o option
[create settings.mk]
[download]    binutils-2.38.tar.bz2
[git-log]   d206e2e6 Document that MXE is used by R.
[download]    ccache-3.6.tar.xz
[download]    mingw-w64-v10.0.0.tar.bz2
[download]    gmp-6.2.1.tar.xz
[download]    isl-0.16.1.tar.bz2
[download]    mpfr-4.1.0.tar.xz
[download]    mpc-1.2.1.tar.gz
[download]    gcc-11.3.0.tar.xz
[download]    pkgconf-da179fd.tar.gz
[download]    libtool-2.4.6.tar.xz
[download]    zlib-1.2.12.tar.xz
[download]    libiconv-1.17.tar.gz
[download]    gettext-0.21.tar.lz
[download]    libunistring-0.9.10.tar.xz
[download]    libidn2-2.3.0.tar.lz
[download]    libtasn1-4.18.0.tar.gz
[download]    nettle-3.8.tar.gz
[download]    gnutls-3.6.16.tar.xz
[check reqs]
[nonet lib]   /build/mxe/usr/x86_64-pc-linux-gnu/lib/nonetwork.so                                                                     
[build]       mxe-conf                x86_64-w64-mingw32.shared
[build]       mxe-conf                x86_64-pc-linux-gnu

Failed to build package mxe-conf for target x86_64-w64-mingw32.shared!
------------------------------------------------------------
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
python --version
bash: line 1: python: command not found
make[1]: *** [Makefile:888: build-only-mxe-conf_x86_64-w64-mingw32.shared] Error 127
make[1]: Leaving directory '/build/mxe'
real    0m1.989s
user    0m1.919s
sys     0m0.083s
------------------------------------------------------------
[log]      /build/mxe/log/mxe-conf_x86_64-w64-mingw32.shared

make: *** [Makefile:887: /build/mxe/usr/x86_64-w64-mingw32.shared/installed/mxe-conf] Error 1
make: *** Waiting for unfinished jobs....

Failed to build package mxe-conf for target x86_64-pc-linux-gnu!
------------------------------------------------------------
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
python --version
bash: line 1: python: command not found
make[1]: *** [Makefile:888: build-only-mxe-conf_x86_64-pc-linux-gnu] Error 127
make[1]: Leaving directory '/build/mxe'
real    0m1.968s
user    0m1.889s
sys     0m0.092s
------------------------------------------------------------
[log]      /build/mxe/log/mxe-conf_x86_64-pc-linux-gnu

make: *** [Makefile:887: /build/mxe/usr/x86_64-pc-linux-gnu/installed/mxe-conf] Error 1
The command '/bin/sh -c cd mxe && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 cc zlib gnutls            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 boost            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 mesa            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 1 JOBS=1 qtbase            && make MXE_TARGETS='x86_64-w64-mingw32.shared' --jobs 2 JOBS=4 qttools                  qtimageformats                  qtmultimedia                  qtquickcontrols                  qtquickcontrols2             && make clean-junk             && rm -rf /build/mxe/src /build/mxe/log /build/mxe/pkg /build/mxe/.git /build/mxe/.ccache' returned a non-zero code: 2
```

--------------

Could it be that Python 2 is indeed still required? If so, could it be re-added to the list of packages here: https://mxe.cc/#requirements-debian

Or is there something else that I am missing?


Kind regards,

Andre

--- a/docs/index.html
+++ b/docs/index.html
@@ -828,6 +828,7 @@ USE_OSGPLUGIN(&lt;plugin2&gt;)
     python3 \
     python3-mako \
     python3-pkg-resources \
+    python-is-python3 \
     ruby \
     sed \
     unzip \

